docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -143,12 +143,56 @@ For example:
 C:\openenclave\build\x64-Debug
 ```
 
+### Building and installing DCAP libraries on Windows (experimental)
+
+Here are the manual steps to build and install DCAP libaries on your Windows VM, an experimental flow at this time
+
+	• Intel PSW 2.3 needs to be installed:
+		○ http://registrationcenter-download.intel.com/akdlm/irc_nas/15369/Intel%20SGX%20PSW%20for%20Windows%20v2.3.100.49777.exe
+		○ This is a self-extracting ZIP executable, and the unpacked PSW_EXE_RS2_and_before\Intel(R)_SGX_Windows_x64_PSW_2.2.100.48339.exe installer needs to be run.
```

Comment:
This is actually PSW 2.2. I think the correct exe name is `Intel(R)_SGX_Windows_x64_PSW_2.3.100.49777.exe`.

Reply:
Thanks for the catch. Will fix.